src/models/methods/baseMethod.ts

```diff
@@ -1,3 +1,5 @@
+import { Request } from '.'
```

Comment:
```suggestion
import type { Request } from '.'
```
Otherwise there's a dependency cycle